_posts/2024-07-30-julian.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title:  "Steering Committee changes"
+date:   2024-07-30
+categories: [announcements]
+author: "Miles Lubin, Carleton Coffrin, Oscar Dowson, and Changhyun Kwon"
+---
+
+Recently, Juan Pablo Vielma notified the [JuMP Steering Committee](/pages/governance/#steering-committee)
+that he is stepping down (effective immediately) from his role on the Steering
+Committee.
+
+Juan Pablo was instrumental in the success of JuMP, supervising Miles Lubin and
+Joey Huchette as Ph.D. students at MIT. Juan Pablo helped organize the first
+three iterations of the JuMP-dev workshop: [JuMP-dev 2017](/meetings/mit2017/) at
+MIT, [JuMP-dev 2018](/meetings/bordeaux2018/) in Bordeaux, and [JuMP-dev 2019](/meetings/santiago2019/)
+in Santiago, Chile. More recently, Juan Pablo was the Principal Investigator of
```

Comment:
I would change "the Principal Investigator" to "a Principal Investigator" (as technically Alan is the PI and I am co-PI).